corehq/apps/app_manager/templates/app_manager/v2/partials/releases.html

```diff
@@ -61,7 +61,7 @@
         var app_langs = {{ app.langs|JSON }};
         var app_profiles = {{ app.build_profiles|JSON }}
 
-        {% if build_profile_access %}
+        {% if not intro_only and build_profile_access %}
```

Comment:
It's when your app is new/empty and instead of the deploy page you get the fancy new help content:
![screen shot 2017-04-27 at 12 13 02 pm](https://cloud.githubusercontent.com/assets/1486591/25493052/ebf55a90-2b42-11e7-8938-c47e47a0a29a.png)